src/routes/PrivateRoute.jsx

```diff
@@ -6,7 +6,6 @@ import { useAuth } from "../context/AuthContext";
 import Sidebar from "../layouts/Sidebar";
 import Navbar from "../layouts/Navbar";
 
```

Comment:
I believe , i told you to hit an endpoint for some data that you've asked, i dont see it here. Not sure, what i've asked.
WIll discuss about this @Parasuramandckap  remind me